### PR TITLE
Create oncotarget.csl

### PR DESCRIPTION
**Pull request by taraonco**

### CODE REVIEW

File: oncotarget.csl [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+      <if type="report">
+        <text variable="number" prefix="Report No.: "/>
+      </if>
+    </choose>
+  </macro>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout prefix="(" suffix=")" delimiter=",">
```
Comment (rmzelle) [inline oncotarget.csl]: Shouldn't this use square brackets?

http://www.impactjournals.com/oncotarget/index.php?journal=oncotarget&page=about says "Cite only the number assigned to the reference: use [ ] not ( )."

Comment (taraonco) [inline oncotarget.csl]: You are correct! I've fixed this.

File: oncotarget.csl [hunk 2]
```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="en-US">
+  <info>
+    <title>Oncotarget</title>
+    <id>http://www.zotero.org/styles/oncotarget</id>
+    <link href="http://www.zotero.org/styles/oncotarget" rel="self"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="http://www.impactjournals.com/oncotarget" rel="documentation"/>
+    <link href="http://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>
```
Comment (rmzelle) [inline oncotarget.csl]: Can you delete `<link href="http://www.nlm.nih.gov/bsd/uniform_requirements.html" rel="documentation"/>` as well?

Comment (taraonco) [inline oncotarget.csl]: Sure!

**Comment by adam3smith** [comment]
Thanks, looking good!

**Pull request merged**